src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -80,6 +81,10 @@ public void setMotor(double armSpeed) {
         this.motorSpeedWidget.setDouble(armSpeed);
     }
 
+    public void setAngle(double angle) {
```

Comment:
we should limit the rotation here so we don't over rotate as well should we continue to do it in periodic, but with the correct limits